src/components/Person/Person.jsx

```diff
@@ -1 +1,35 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => {
+  const {
+    name,
+    age,
+    sex,
+    isMarried,
+    partnerName,
+  } = person;
+
+  const partner = (sex === 'm')
+    ? 'wife'
+    : 'husband';
+
+  return (
+    <>
+      <section className="Person">
+        <h2 className="Person__name">
+          {`My name is ${name}`}
+        </h2>
+
+        {age && (
+          <p className="Person__age">{`I am ${age}`}</p>
+        )}
+
+        <p className="Person__partner">
+          {(isMarried)
+            ? `${partnerName} is my ${partner}` : `I am not married`}
```

Comment:
Each part of the ternary operator should be placed at the beginning of a new line and maybe you can put this logic in additional variable